python_coreml_stable_diffusion/torch2coreml.py

```diff
@@ -483,35 +468,19 @@ def convert_vae_encoder(pipe, args):
         raise RuntimeError(
             "convert_unet() deletes pipe.unet to save RAM. "
             "Please use convert_vae_encoder() before convert_unet()")
-
-    sample_shape = (
+    
+    height = (args.latent_h * 8) if args.latent_h else pipe.vae.config.sample_size
```

Comment:
This new shape logic breaks for [stabilityai/stable-diffusion-2-1-base](https://huggingface.co/stabilityai/stable-diffusion-2-1-base/blob/main/vae/config.json) where `pipe.vae.config.sample_size = 768` (non-base version) while the current shape logic of `(args.latent_h/w or pipe.unet.config.sample_size) * 8 = 512` is accurate. Could we please keep that logic?

Reply:
@atiorh Thanks for the review! I thought that model output 768x768 images by default and so the encoder would take that size too, I’m not currently at home so I’m not able to test this but I have reverted the size calculation to what you mention.